### PR TITLE
Add OpenVINO int8 Python API

### PR DESCRIPTION
**Pull request by qiyuangong**
Add int8 Python API.

### CODE REVIEW

**Comment by qiyuangong** [comment]
Jenkins passed http://10.239.47.211:18888/job/ZOO-PR-Validation/2324/

**Comment by glorysdj** [comment]
LGTM

File: docs/docs/APIGuide/PipelineAPI/inference.md [hunk 1]
```diff
@@ -242,6 +242,12 @@ List<List<JTensor>> predictOutput = model.predictInt8(predictInput);
 val predictOutput = model.doPredictInt8(predictInput)
 ```
 
+**Python**
+
+```scala
```
Comment (hkvision) [inline docs/docs/APIGuide/PipelineAPI/inference.md]: should be python...

File: pyzoo/zoo/pipeline/inference/inference_model.py [hunk 2]
```diff
@@ -61,6 +61,17 @@ def load_openvino(self, model_path, weight_path):
         callBigDlFunc(self.bigdl_type, "inferenceModelLoadOpenVINO",
                       self.value, model_path, weight_path)
 
+    def load_openvino_int8(self, model_path, weight_path, batch_size):
```
Comment (hkvision) [inline pyzoo/zoo/pipeline/inference/inference_model.py]: is this covered in ut?

Comment (qiyuangong) [reply pyzoo/zoo/pipeline/inference/inference_model.py]: Yes. Covered in `test_inference_model.py`.

File: docs/docs/APIGuide/PipelineAPI/inference.md (hunk 1 above)
Comment (qiyuangong) [reply docs/docs/APIGuide/PipelineAPI/inference.md]: Fixed. Thanks :)

**Pull request merged**